scripts/generate.py

```diff
@@ -74,6 +74,21 @@ def add_frozen_crss(crss):
             crss = [*crss, *dom]
     return crss
 
+def make_projjson_index(dest_dir, crss):
+    dest_file = f'{dest_dir}/projjson_index.json'
+
+    index = [
+        {
+            "auth_name": crs["auth_name"],
+            "code": crs["code"],
+            "link": f"./ref/{crs['auth_name'].lower()}/{crs['code'].lower()}/projjson.json",
```

Comment:
```suggestion
            "link": f"./ref/{crs['auth_name'].lower()}/{crs['code']}/projjson.json",
```

Reply:
The code is not lowercase. See that https://sr-org-scratch.github.io/ref/ogc/crs84/projjson.json returns a 404
There are some authorities that use letters in their codes, like OGC or IGNF.
(The authority is lowercase to keep backwards compatibility in a 18 years old project).